subsys/bootloader/bootloader.h

```diff
@@ -0,0 +1,72 @@
+/*TODO: Remove when multi-image has been merged*/
```

Comment:
Why not put the addresses in Kconfig? That would make them available in CMake as well.

Reply:
The only reasoning was less diff from now to multi image. Since the variable names will be prefixed with CONFIG_ instead of DT_. 

This is a fixup until multi-image is merged when multi image is merged removing `bootloader.h` would give you compile errors so it's easier to remove where it's used. Instead of modifying the Kconfig file then revert. All files touched but it's open for discussion if you would rather prefer it placed in Kconfig.

Reply:
Ok, I was wondering if we could reuse the values for the provision override parameters in the cmake file. It's maybe not worth it at this point, at least if it will change again.

Reply:
I don't think it's worth it at this point, as we would have to change it again. So it's better to change it when we have the actual build system in place :1st_place_medal: